### PR TITLE
add step to convert raw to mzml in experimental design

### LINKED ISSUE #107: in conversion process: convert .raw to .mzML in experimental design
The tool is strict regarding the file type so this will trigger an error.

### PR DESCRIPTION
**Pull request by jpfeuffer**
fixes #107 
You can't do it in the conversion process since you cannot update the tsv file one by one.
I would love to use groovy or something to not spawn another process but I just cannot make it work.

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Fixed the branch protection already. Seems like it does not rerun though.

File: main.nf [hunk 1]
```diff
@@ -308,7 +308,27 @@ if (params.expdesign)
...
         .fromPath(params.expdesign)
-        .set { ch_expdesign }
+        .set { ch_expdesign_pre }
+
+    process expdesign_raw2mzml {
+
+    label 'process_very_low'
+    label 'process_single_thread'
+
+    echo true
```
Comment (Zethson) [inline main.nf:318]: Is it really required here?

Or was this just for you to debug?

Comment (jpfeuffer) [reply main.nf:318]: Ah thanks yes debug

Comment (Zethson) [reply main.nf:318]: NP. Feel free to merge afterwards. LGTM.

**Pull request merged**